code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -433,9 +487,10 @@
 
 	X.face_atom(A) //Face towards the target so we don't look silly
 
-	if(!line_of_sight(X, A)) //Need line of sight.
-		to_chat(X, span_xenowarning("We lost line of sight to the target!"))
-		return fail_activate()
+	if(require_los)
+		if(!line_of_sight(X, A)) //Need line of sight.
+			to_chat(X, span_xenowarning("We lost line of sight to the target!"))
+			return fail_activate()
```

Comment:
AFAIK this check is actually entirely redundant, since can_use_ability is immediately checked before use_ability.